src/shogun/neuralnets/ConvolutionalFeatureMap.cpp

```diff
@@ -283,6 +287,32 @@ void CConvolutionalFeatureMap::convolve(
 			outputs.matrix+i*outputs.num_rows + outputs_row_offset,
 			m_output_height, m_output_width, false);
 
+#if defined(HAVE_LINALG_LIB)
```

Comment:
Using gist for the wiki thing?

On Wed, Dec 23, 2015, 17:15 Heiko Strathmann <notifications@github.com>
wrote:

> In src/shogun/neuralnets/ConvolutionalFeatureMap.cpp
> <https://github.com/shogun-toolbox/shogun/pull/2961#discussion_r48383563>:
>
> > @@ -283,6 +287,32 @@ void CConvolutionalFeatureMap::convolve(
> >  			outputs.matrix+i*outputs.num_rows + outputs_row_offset,
> >  			m_output_height, m_output_width, false);
> >
> > +#if defined(HAVE_LINALG_LIB)
>
> I would go for the more efficient one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shogun-toolbox/shogun/pull/2961/files#r48383563>.
>